docs/89-NEWS.md

```diff
@@ -0,0 +1,839 @@
+# NEWS
+
+![Logo](../../images/icons/aw_64_shadow.png)
```

Comment:
The API doc can e.g. also be installed to `/usr/share/doc/awesome` (I think Debian does something like this?), so yeah, please add this file.